scipy/special/_spherical_bessel.py

```diff
@@ -155,7 +267,8 @@ def spherical_in(n, z, derivative=False):
 
 
 def spherical_kn(n, z, derivative=False):
-    r"""Modified spherical Bessel function of the second kind or its derivative.
+    r"""Modified spherical Bessel function of the second kind or its
```

Comment:
Better to leave this as a single line per PEP 257:

https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
Sure thing!